pkg/internal/common/mockoidc/provider.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package mockoidc
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+// StartProvider a new http endpoint serving a mock OpenID Connect configuration file and returns its base URL.
```

Comment:
Do you mean "StartProvider starts a new http endpoint..."?

Reply:
@DRK3 fixed